### PR TITLE
JTL-SoftwareGmbH.JTL-Wawi version 1.7.8.1

### PR DESCRIPTION
**Pull request by AndyHeinz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98608)

### CODE REVIEW

**Comment by AndyHeinz** [comment]
@microsoft-github-policy-service agree

**Comment by AndyHeinz** [comment]
looks good

**Comment by PeterLustig21** [comment]
Approved

**Comment by PeterLustig21** [comment]
Approved

**Review by PeterLustig21 — APPROVED** [review]
GO

**Review by PeterLustig21 — DISMISSED** [review]
Checked

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.installer.yaml:4]:
```suggestion
PackageIdentifier: JTL.Wawi
```
Please add this under the existing directory https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JTL/Wawi

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: JTL.Wawi
```
Please add this under the existing directory https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JTL/Wawi

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.yaml:4]:
```suggestion
PackageIdentifier: JTL.Wawi
```
Please add this under the existing directory https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JTL/Wawi

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.installer.yaml:5]:
```suggestion
PackageVersion: 1.7.8.1
Scope: machine
```
Package installs in machine scope

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
+PackageLocale: en-US
+Publisher: JTL-Software GmbH
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml:7]:
```suggestion
Publisher: JTL-Software-GmbH
```
Must match Control Panel / Apps&Features

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
+PackageLocale: en-US
+Publisher: JTL-Software GmbH
+PackageName: JTL-Wawi
+License: FREE
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
+PackageLocale: en-US
+Publisher: JTL-Software GmbH
+PackageName: JTL-Wawi
+License: FREE
+ShortDescription: JTL-Wawi Setup
```
Comment (mdanish-kh) [inline manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml:10]:
```suggestion
ShortDescription: The free inventory management system noticeably simplifies your business processes. Transparent flow of goods, better overview, full control.
```
Short description must give a description of the package

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please find the suggestions below.

After committing the suggested changes below, you would have to move the version folder `manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1` to `manifests/j/JTL/Wawi` and also rename the .yaml files to match the PackageId. (i.e., JTL-SoftwareGmbH.JTL.Wavi.installer.yaml -> JTL.Wawi.installer.yaml and so on)


One way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes. The directory structure should look similar to



After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

If you find this all too tedious, you could just close this PR and make a new one with wingetcreate for `JTL.Wavi`

**Review by mdanish-kh — COMMENTED** [review]
Please also add the .yaml file for the German locale that exists in previous manifests
https://github.com/microsoft/winget-pkgs/blob/master/manifests/j/JTL/Wawi/1.5.49.0/JTL.Wawi.locale.de-DE.yaml

**Comment by AndyHeinz** [comment]
Hi,
to me the filenames look the same:
grafik

How do I resolve this?
Thanks!

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**